Loop/Managers/LoopDataManager.swift

```diff
@@ -126,6 +126,19 @@ final class LoopDataManager {
 
                     self.insulinEffect = nil
 
+                    self.notify(forChange: .bolus)
+                }
+            },
+            NotificationCenter.default.addObserver(
+                forName: .InsulinDeliveryStoreCacheDidChange,
+                object: doseStore.insulinDeliveryStore,
+                queue: OperationQueue.main
+            ) { (note) in
+                self.dataAccessQueue.async {
+                    self.logger.default("Received notification of cached dosing changing")
+
+                    self.insulinEffect = nil
```

Comment:
I wasn't really sure about this line, but it's what the other bolus change update was doing so I figured it was probably necessary 🤷‍♂

Reply:
This is necessary to trigger a recalc of insulin effect.

Reply:
Thanks for the clarification!